riotfile.py

```diff
@@ -1478,7 +1478,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 Venv(
                     pys=["2.7"],
                     # pytest==4.6 is last to support python 2.7
-                    pkgs={"pytest": ">=4.0,<4.6", "msgpack": latest},
+                    pkgs={"pytest": ">=4.0,<4.6", "asynctest": "==0.11.1", "msgpack": latest},
```

Comment:
It might be better to keep these test environments as is but add another for running a regression for asynctest in particular.

Reply:
:+1: